src/form/v2/birth/dedupConfig.ts

```diff
@@ -15,6 +15,13 @@ const childDobWithin5Days = field('child.dob').dateRangeMatches({ days: 5 })
 const similarNamedMother = field('mother.name').fuzzyMatches()
 const similarAgedMother = field('mother.dob').dateRangeMatches({ days: 365 })
 const sameMotherNid = field('mother.nid').strictMatches()
+const sameMotherPassport = field('mother.passport').strictMatches()
+const sameMotherBrn = field('mother.brn').strictMatches()
+const sameMotherIdentifier = or(
+  sameMotherNid,
+  sameMotherPassport,
+  sameMotherBrn
+)
```

Comment:
Really minor suggestion, as is is completely fine too :)

```suggestion
const sameMotherIdentifier = or(
  field('mother.nid').strictMatches(),
  field('mother.passport').strictMatches(),
  field('mother.brn').strictMatches()
)
```